### ISSUE TITLE
Requesting new versions of AOM AV1 and vpx?

### ISSUE THREAD

**Issue by mayconhvab1** [BtbN/FFmpeg-Builds#342]
Would you like to know how to request new versions of audio and video codecs that may be released in the future?

I noticed that BtbN said in the thread: https://github.com/BtbN/FFmpeg-Builds/issues/341 that you will not be able to add new patches.

But how can I contribute to FFmpeg since I don't know how to program, much less know which path to follow to pass on information about new codecs and recently released versions?

I apologize if I'm not following BtbN's order, as I don't know and I can't request a new pull request.

It appears that the FFmpeg BtbN pull request is not accepting my request or I may have to do some other way of requesting features and adding new codec information or other FFmpeg BtbN information to github.

I would like someone to guide me please.

I understand, but I really don't know and I can't request a pull request.

In short, I request new versions of: AOM AV1 (3.8.1) and vpx (1.14.0).

Give me information on how to request a new pull request, but remember that I'm not a programmer and unfortunately I don't know how to program.

Thanks so far.

**Comment by BtbN** [comment]
I just updated all dependencies a day or two ago. Are you sure they're already outdated again?
If you are suggesting adding a new codec to ffmpeg itself, you will need to direct that request upstream.

**Comment by mayconhvab1** [comment]
OK.

I apologize for the misunderstanding, but I thought you had received some information in the pull request about requesting or adding new versions of these two codecs.

No, fortunately they are all up to date, I just haven't received any information that they are up to date.

But still, thank you very much.
